cpp/CMakeLists.txt

```diff
@@ -146,6 +146,7 @@ include(${rapids-cmake-dir}/cpm/cuco.cmake)
 rapids_cpm_cuco(BUILD_EXPORT_SET cugraph-exports)
 
 include(cmake/thirdparty/get_raft.cmake)
+find_package(raft COMPONENTS distance)
```

Comment:
`get_raft.cmake` should be doing this now so there shouldn't be a need for this line

Reply:
Fixed